First merge attempt

SMAPPER opened the pull request.

philhagen commented:
It looks like alexa_update.sh has an unresolved merge.  All others merged cleanly.  Note that I changed the path and filename for the dictionaries to the new locations and updated the configuration files to match.  I believe this should be a clean and complete merge but please confirm before I close the merge

SMAPPER commented:
For now you can ignore the alexa_update.sh script.  Currently it is a
roughneck way of downloading the alexa top one million and having logstash
shove it into an Elasticsearch index.  With either Mark's API or if you
create a Logstash filter this will become obsolete.  I say we hold off
until I get the final copy of the script from Mark or if you are able to
develop a logstash filter for it.


Sincerely,

Justin Henderson
(312) 857-5755
Systems and Security Architect
GSE # 108, Cyber Guardian Red / Blue
http://www.linkedin.com/in/justinhenderson2014/


On Sun, Nov 13, 2016 at 4:55 PM, Phil Hagen <notifications@github.com>
wrote:

> It looks like alexa_update.sh has an unresolved merge. All others merged
> cleanly. Note that I changed the path and filename for the dictionaries to
> the new locations and updated the configuration files to match. I believe
> this should be a clean and complete merge but please confirm before I close
> the merge
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/philhagen/sof-elk/pull/48#issuecomment-260219357>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ALPAz5t2PZVDppv1AMXqmapaTGRFm9LFks5q95VNgaJpZM4KY2jM>
> .
>

philhagen commented:
Roger - I'm going to remove it from the tree for now - we can re-add if needed
